Makefile

```
@@ -75,6 +75,9 @@ helm-push: helm-package
 	@if [ ! $(REGISTRY_IS_OCI) ]; then \
 	    repo_flag="--repo"; \
 	fi; \
+        if [ $(REGISTRY_PLAIN_HTTP) = "true" ];   then \
+            plain_http_flag="--plain-http"; \
+        fi; \
 	for chart in $(CHARTS_PACKAGE_DIR)/*.tgz; do \
```

Indentation fix AND always setting this variable to avoid using undefined vars:
```suggestion
	fi; \
	if [ $(REGISTRY_PLAIN_HTTP) = "true" ]; \
	then plain_http_flag="--plain-http"; \
	else plain_http_flag=""; \
	fi; \
	for chart in $(CHARTS_PACKAGE_DIR)/*.tgz; do \
```